src/main/java/com/epam/test_generator/services/SuitService.java

```diff
@@ -61,4 +67,31 @@ public void removeSuit(long suitId) {
 
         caseVersionDAO.delete(suit.getCases());
     }
+
+    public void updateSuitRowNumber(List<SuitRowNumberUpdateDTO> rowNumberUpdates) {
+        if (rowNumberUpdates.isEmpty()) {
```

Comment:
Is it really a error scenario? Check it please.

Reply:
It isn't, but I would call it validation